examples/teaching-vm_openrefine/Dockerfile.original

```diff
@@ -0,0 +1,27 @@
+#docker build --rm -t psychemedia/ou-tm351-openrefine-test .
+#docker build --rm --build-arg BASE=etc -t psychemedia/ou-tm351-openrefine-test .
+
+ARG BASE=psychemedia/ou-tm351-minimal-test
+# Base machine
+FROM ${BASE}
+
+MAINTAINER tony.hirst@gmail.com
+
+ENV DOCKERBUILD 1
+
+
+#Build OpenRefine
+ADD openrefine.sh /tmp/openrefine/openrefine.sh
+ADD root /tmp/openrefine/root
+
+RUN /tmp/openrefine/openrefine.sh
```

Comment:
Don’t we champion not deleting original source code?

Reply:
We don't champion anything in the `.original` file. That is the file that is bad and I'm trying to improve.